.github/workflows/deps.yaml

```diff
@@ -167,7 +179,121 @@ jobs:
         with:
           name: linux_arm64_substrait
           path: deps/linux_arm64
+      - uses: actions/download-artifact@v4
+        with:
+          name: iceberg-v1.1.3-extension-linux_arm64
+          path: deps/linux_arm64
       - name: Push static libraries
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: Re-build static libraries
+
+  iceberg:
+    name: Build extension binaries
+    uses: ./.github/workflows/iceberg_extension_distribution.yaml
+    with:
+      extension_name: iceberg
+      duckdb_version: v1.1.3
+      ci_tools_version: v1.1.3
+      exclude_archs: 'linux_amd64_gcc4'
+
+  test_darwin_amd64:
+    runs-on: macos-13
```

Comment:
how do we know which arch we are using? I think macos-13 is x64 only and the two darwin builds are using the same arch

Reply:
You are right Ryan. Also for linux-20.04, I couldn't directly make it work on a arm64 runner (didn't like the approach of using QEMU as it is all docker and couldn't fit into matrix)